### PR TITLE
Explain about `lounge` command in dev installations

### PR DESCRIPTION
**Pull request by drkitty**
The readme was missing some info about using the development version, so I fixed it. I hope that's okay.

### CODE REVIEW

**Comment by astorije** [comment]
Mmh, odd. `npm start` should just work. `npm start command` or `npm start -- command` should let you run the subcommands of the `lounge` CLI.

**Comment by drkitty** [comment]
Crap, you're right! Sorry about that. I knew `npm start` worked, but I didn't know you could pass it args. Should've asked in chat first.

File: README.md [hunk 1]
```diff
@@ -50,9 +50,10 @@ The following commands install the development version of The Lounge:
 git clone https://github.com/thelounge/lounge.git
 cd lounge
 npm install
-npm start
```
Comment (astorije) [inline README.md]: I wouldn't remove this line, and your next change could only be about running subcommands.

**Comment by drkitty** [comment]
Want me to squash this?

**Comment by YaManicKill** [comment]
> Want me to squash this?

Yes please.

**Comment by drkitty** [comment]
I squashed (and rebased while I was at it). I guess you don't get a notification when I push, so I should have added a comment. My bad.

**Comment by YaManicKill** [comment]
@drkitty Yeah, we don't get notifications about changes to code. Only comments. Thanks :-)

**Pull request merged**